site/docs/master/httpproxy.md

```diff
@@ -1263,6 +1263,34 @@ spec:
             subjectName: foo.marketing
 ```
 
+## Client Certificate Validation
+
+It is possible to protect the backend service from unauthorized external clients by requiring the client to present a valid TLS certificate.
+Envoy will validate the client certificate by verifying that it is not expired and that a a chain of trust can be established to the configured trusted root CA certificate.
+Only those requests with a valid client certificate will be accepted and forwarded to the backend service.
+
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: with-client-auth
+spec:
+  virtualhost:
+    fqdn: www.example.com
+    tls:
+      secretName: secret
+      clientValidation:
+        caSecret: client-root-ca
+  routes:
+    - services:
+        - name: s1
+          port: 80
+```
+
+The preceding example enables validation by setting the optional `clientValidation` attribute.
+Its mandatory attribute `caSecret` contains a name of existing Kubernetes Secret that must be of type "Opaque" and have a data key named as `ca.crt`.
```

Comment:
```suggestion
Its mandatory attribute `caSecret` contains a name of an existing Kubernetes Secret that must be of type "Opaque" and have a data key named `ca.crt`.
```

Reply:
Thanks, fixed!  I appreciate having native speaker go through the text :)